package.json

```diff
@@ -77,10 +77,8 @@
   "keywords": [
     "css",
     "features",
-    "list",
     "specifications",
-    "stages",
-    "tc39"
```

Comment:
This was a bit weird and suddenly caught my attention.
`cssdb` is completely unrelated to `tc39`.